app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from abc import ABC
```

Comment:
Add one blank line between different import groups to follow PEP-8 here and in the `elf.py`